app/src/src/components/common/FormatDate/toYearMonthDayInNumbers.jsx

```diff
@@ -1,15 +1,11 @@
 // takes in Date and returns the date in YYYY-MM-DD format (ex: 1960-01-01)
 const toYearMonthDayInNumbers = ({ selectedValue }) => {
-  const currYear = selectedValue.getFullYear().toString();
-  const currMonth =
-    selectedValue.getMonth() < 10
-      ? ('0' + (selectedValue.getMonth() + 1)).toString().slice(-2)
-      : (selectedValue.getMonth() + 1).toString().slice(-2);
-  const currDay =
-    selectedValue.getDate() < 10
-      ? ('0' + selectedValue.getDate()).toString().slice(-2)
-      : selectedValue.getDate().toString().slice(-2);
-  const YYYYMMDD = currYear + '-' + currMonth + '-' + currDay;
+  const YYYYMMDD = selectedValue.toLocaleDateString('en-CA', {
```

Comment:
there was a comment before saying that the toLocaleDateString was returning the date formatting with slashes (MM/DD/YY), but it was returning in YYYY-MM-DD format for me so I don't know

Reply:
@d36mehta  There might be some configuration settings that you can tell the toLocaleDateString to use when formatting. Have you checked that out yet?

Reply:
I think the comment above the old formatDate function in the CreateCommunityScreen file was saying that the date was supposed to return in YYYY-MM-DD format so it can be saved, but instead returned in MM/DD/YY format (different than what the MDN docs says). So the old formatDate function was manually trying to turn MM/DD/YY into YYYY-MM-DD, so it can save it. But it's now already returning as YYYY-MM-DD, so I guess it's returning like what the MDN doc says now

Reply:
Sounds good!